agu_oss/clean_data.py

```diff
@@ -1,5 +1,5 @@
 """A second sample module to clean data."""
-
+    # Does it work? -J
```

Comment:
it sure does  @kjhttr !! great job submitting this pull request! your PR ran successfully as well!